src/controllers/presenter.controller.ts

```diff
@@ -29,26 +25,14 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-export const post = async (req: Request, res: Response, next: NextFunction) => {
+export const post = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Haven't you got to do an `await saveAndContinue(req, session);` in this method now/still?

Reply:
Good spot, I will add it